pkg/compose/create.go

```diff
@@ -595,8 +595,11 @@ func setLimits(limits *types.Resource, resources *container.Resources) {
 		resources.Memory = int64(limits.MemoryBytes)
 	}
 	if limits.NanoCPUs != "" {
-		i, _ := strconv.ParseInt(limits.NanoCPUs, 10, 64)
-		resources.NanoCPUs = i
+		f, _ := strconv.ParseFloat(limits.NanoCPUs, 64)
+		resources.NanoCPUs = int64(f * 1e9)
```

Comment:
No right that's seems legit to assign only if there is no error 👍 
Otherwise the `NanoCPUs` is coming from the [Docker API](https://github.com/moby/moby/blob/20d6b5c1bdedffe74f9dd774e3e22ad59360a51c/api/types/container/host_config.go#L360) and is different from the `CPUs` property of Compose services
And the Compose specification define `deploy.resources.limits.cpus` as [the number of cpus](https://github.com/compose-spec/compose-spec/blob/master/deploy.md#cpus), so user should define them like `0.5` or `2.4` if they want